packages/backend-output-schemas/src/index.ts

```diff
@@ -84,6 +85,20 @@ export * from './function/index.js';
  */
 export const functionOutputKey = 'AWS::Amplify::Function';
 
+/**
+ * ---------- AI conversation exports ----------
+ */
+
+/**
+ * re-export the AI conversation output schema
+ */
+export * from './ai/conversation/index.js';
+
+/**
+ * Expected key that AI conversation output is stored under
+ */
+export const aiConversationOutputKey = 'AWS::Amplify::AI::Conversation';
```

Comment:
the naming here follows `ai-constructs/<sub_usecase>` 
I.e. if we add another AI feature not related to conversation we'd have `AWS::Amplify::AI::SomethingNew`